src/app/wizard/step/provider-settings/provider/extended/kubevirt/template.html

```diff
@@ -94,23 +106,18 @@
                        [name]="Controls.PreAllocatedDataVolumeUrl + i"
                        type="text"
                        autocomplete="off">
-                <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError( 'required')">
+                <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError('required')">
                   <strong>Required</strong>
                 </mat-error>
+                <mat-error *ngIf="control.get(Controls.PreAllocatedDataVolumeUrl).hasError('pattern')">
```

Comment:
Can we use `type="url"` instead of custom pattern validator?

Reply:
I need to show proper error message too, while using `type="url"` there is no errors in the form control
<img width="555" alt="image" src="https://user-images.githubusercontent.com/56511988/199955256-0907f3da-7031-4716-b493-88e71fc31c93.png">